Source/DafnyCore/AST/Modules/ModuleDefinition.cs

```diff
@@ -73,12 +73,18 @@ public class ModuleDefinition : RangeNode, IDeclarationOrUsage, IAttributeBearin
   [FilledInDuringResolution]
   public readonly Graph<ICallable> CallGraph = new();
 
+  // This field is only populated if `defaultFunctionOpacity` is set to something other than transparent
   [FilledInDuringResolution]
   public readonly Graph<ICallable> InterModuleCallGraph = new();
 
   [FilledInDuringResolution]
   public int Height;  // height in the topological sorting of modules;
 
+  /// <summary>
+  /// The following class stores the relative name of any Declaration that is reachable from this module
+  /// as list of NameSegments, along with a flag for if the Declaration is revealed or merely provided.
+  /// For example, if "A" is a module, a function "A.f()" will be stored as f ~> {"A", "_default"} in the AccessibleMembers field.
```

Comment:
What does that syntax `f ~> {"A", "_default"}` mean? Please explain.

Reply:
AccessibleMembers is a `Dictionary<Declarations, AccessibleMember>` By `f ~> {"A", "_default"}` I meant that in this dictionary, `f` will be mapped to the AccessibleMember whose AccessPath list contains (the NameSegment versions of) "A" and "_default". I guess I should write it this verbosely in the comment itself?

Reply:
Done.